src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/util/CustomTimestampSerialiser.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.util;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+
+import java.io.IOException;
+import java.sql.Timestamp;
+
+public class CustomTimestampSerialiser extends StdSerializer<Timestamp> {
```

Comment:
Could you please add a java doc comment on this class with an example of date format ?

Reply:
even better, a unit test :P